### ISSUE TITLE
No vario sound 

### ISSUE THREAD

**Issue by georghg** [fhorinek/BB#227]
Is there a secret mute button on the device? The vario doesn‘t make a single beep since some time on many flights (despite having lift also showed on the display). It is not a problem of auto-start since I put it on always on after experiencing the issue. Also not a hardware issue of the speakers since sometimes it does beep.

first I thought it was an issue of fw242 and 243 however, the problem also showed up on fw232 now. Any suggestions?

Kind regards
Georg

**Comment by fhorinek** [comment]
No there is no mute button, can you hear bluetooth audio?

**Comment by georghg** [comment]
Yes, I tested bluetooth audio and that worked. Therefore, I concluded that it is not a hardware issue.

I did a full wipe now and installed 232. Did some 10-20 starts of the device and vario sound was always on now. Let‘s see if it stays like this.

Before the vario sound issue showed up I switched between 232, 242 and 243 in both directions (up- and downgrading). Maybe that caused the issue. Could at least explain why the full wipe seems to have solved it now. 

I will stay in 232 a while now an observe whether it shows up again. If so, I will let you know.

**Comment by fhorinek** [comment]
Thank you for the report.
